### ISSUE TITLE
segment fault on libleveldb.so when load data

### ISSUE THREAD

**Issue by ivysoftware** [piiswrong/dec#5]
Hi Junyuan,
Basic, I downloaded two datasets: mnist and reuters by  mnist(reuters)/get_data.sh then pyhton make_mnist(reuters)_data.py.
However, things is not so easy as I thought.
       1. Parameter initialization Problem:  Claimed " You can use dec/pretrain.py to train your own autoencoder. " on the https://github.com/piiswrong/dec 
            but in my practice, python pretrain.py came out of "segment fault on libleveldb.so when load data", which data is default on DB=mnist, and  download by mnist/get_data.sh then pyhton make_mnist_data.py  
     2.  Two(mnist/reuters10k) datasets are segmented fault on training using dec.py. 
     3. Since reutersidf is ok on training, I am looking forward to seeing the amazing result, but the acc is only about 0.35 not a impressive promance of 0.7, during the experiment procedure I assumed the reutersidf data is pretrained well. 
        Could you double check on if datesets is pretrained well ? Although, the loss is very low. 
    4. I have a confuse on the training process: during diff iteration, diff training data is loaded by seek operator, after the current iteration training(each has a 20 times caffe iterations), init weight and init.model are updated, then a fine-tuning of next iteration diff data is processed, is it right?

**Comment by ankitrbansal** [comment]
@ivysoftware  Hi, I am facing the same issues when I try it with mnist dataset. I get a Segmentation Fault on "Opening leveldb mnist_total". Have you found a way to make it work with mnist? Thanks!
